grc/core/generator/flow_graph.tmpl

```diff
@@ -336,18 +336,18 @@ $short_id#slurp
 
 
 def argument_parser():
-    #set $desc_args = 'usage="%prog: [options]", option_class=eng_option'
     #if $flow_graph.get_option('description')
-        #set $desc_args += ', description=description'
     description = $repr($flow_graph.get_option('description'))
+    #else
+    description = None
     #end if
-    parser = OptionParser($desc_args)
+    parser = ArgumentParser(description=description)
```

Comment:
Please don't change the structure here. We only want to emit arguments we really need. That is why the arguments to ArgumentParser() are first collected in a string.